src/main/java/core/basesyntax/service/impl/OperationStrategyImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.strategy.OperationHendler;
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy {
```

Comment:
move this class to strategy package, you don't need to create interface not needed in this case, because it is not a service.